src/Connection/Aggregate/RedisCluster.php

```diff
@@ -276,11 +280,11 @@ protected function guessNode($slot)
      */
     protected function createConnection($connectionID)
     {
-        $separator = strrpos($connectionID, ':');
+        list($host,$port) = explode(':', $connectionID, 2);
```

Comment:
Ah, that does make sense.  The strrpos() did look a little strange.  My main improvement was just to replace blind array accesses w/ the multiple assignment.  I try to always use symbolic names instead of array indexes to make the code easier to read.  How about a little trickery to ensure only the rightmost ':' is split on:

```
  list($port, $host) = array_map('strrev', explode(':', strrev($connectionID), 2));
```
Just a little string reversal, so that a connection id like e.g. '::1:6380' would first get reversed to '0836:1::', then split into two elements around the first colon:  array('0836','1::').  Then the elements reversed again into array('6380','::1'), and the results assigned to variables.

Somehow those substr() and strrpos() calls are bugging me more than the double-reversal above.  What say you?